chips/msp432/src/pcm.rs

```diff
@@ -0,0 +1,188 @@
+        LOCKBKUP OFFSET(1) NUMBITS(1) [
+            /// Backup domain configuration defaults to reset condition
+            BackupDomainConfigurationDefaultsToResetCondition = 0,
+            /// Backup domain configuration remains locked during LPM3.5 entry and exit
+            BackupDomainConfigurationRemainsLockedDuringLPM35EntryAndExit = 1
+        ],
+        /// Force LPM entry
+        FORCE_LPM_ENTRY OFFSET(2) NUMBITS(1) [
+            /// PCM aborts LPM3/LPMx.5 transition if the active clock configuration does not mee
+            FORCE_LPM_ENTRY_0 = 0,
+            /// PCM enters LPM3/LPMx.5 after shuting off the clocks forcefully. Application need
+            FORCE_LPM_ENTRY_1 = 1
+        ],
+        /// Power mode request busy flag
+        PMR_BUSY OFFSET(8) NUMBITS(1) [],
+        /// PCM key
+        PCMKEY OFFSET(16) NUMBITS(16) []
+    ],
+    PCMIE [
+        /// LPM invalid transition interrupt enable
+        LPM_INVALID_TR_IE OFFSET(0) NUMBITS(1) [
+            /// Disabled
+            Disabled = 0,
+            /// Enabled
+            Enabled = 1
+        ],
+        /// LPM invalid clock interrupt enable
+        LPM_INVALID_CLK_IE OFFSET(1) NUMBITS(1) [
+            /// Disabled
+            Disabled = 0,
+            /// Enabled
+            Enabled = 1
+        ],
+        /// Active mode invalid transition interrupt enable
+        AM_INVALID_TR_IE OFFSET(2) NUMBITS(1) [
+            /// Disabled
+            Disabled = 0,
+            /// Enabled
+            Enabled = 1
+        ],
+        /// DC-DC error interrupt enable
+        DCDC_ERROR_IE OFFSET(6) NUMBITS(1) [
+            /// Disabled
+            Disabled = 0,
+            /// Enabled
+            Enabled = 1
+        ]
+    ],
+    PCMIFG [
+        /// LPM invalid transition flag
+        LPM_INVALID_TR_IFG OFFSET(0) NUMBITS(1) [],
+        /// LPM invalid clock flag
+        LPM_INVALID_CLK_IFG OFFSET(1) NUMBITS(1) [],
+        /// Active mode invalid transition flag
+        AM_INVALID_TR_IFG OFFSET(2) NUMBITS(1) [],
+        /// DC-DC error flag
+        DCDC_ERROR_IFG OFFSET(6) NUMBITS(1) []
+    ],
+    PCMCLRIFG [
+        /// Clear LPM invalid transition flag
+        CLR_LPM_INVALID_TR_IFG OFFSET(0) NUMBITS(1) [],
+        /// Clear LPM invalid clock flag
+        CLR_LPM_INVALID_CLK_IFG OFFSET(1) NUMBITS(1) [],
+        /// Clear active mode invalid transition flag
+        CLR_AM_INVALID_TR_IFG OFFSET(2) NUMBITS(1) [],
+        /// Clear DC-DC error flag
+        CLR_DCDC_ERROR_IFG OFFSET(6) NUMBITS(1) []
+    ]
+];
+
+pub struct Pcm {
+    registers: StaticRef<PcmRegisters>,
+}
+
+impl Pcm {
+    const fn new() -> Pcm {
+        Pcm {
+            registers: PCM_BASE,
+        }
+    }
+    // currently not sure about the interface, so just implement a simple
+    // method for activating AM_LDO_VCORE1 to provide enough power for 48MHz
+    pub fn set_high_power(&self) {
+        while self.registers.ctl1.is_set(PCMCTL1::PMR_BUSY) {}
```

Comment:
:/, another one for the #1552 wishlist

(nothing for you to do here; just want to flag it on that issue)